app/src/main/java/com/example/faunaapp/repository/Repository.java

```diff
@@ -0,0 +1,80 @@
+package com.example.faunaapp.repository;
+
+import android.os.AsyncTask;
+import android.os.Handler;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.ApolloClient;
+import com.apollographql.apollo.api.Response;
+import com.apollographql.apollo.exception.ApolloException;
+import com.faunaapp.graphql.LogInMutation;
+
+import org.jetbrains.annotations.NotNull;
+
+
+public class Repository {
+    private static Repository instance;
+    private LogInAsyncTask logInAsyncTask;
+    private static ApolloClient apolloClient;
+
+    public Repository() {
+        apolloClient = ApolloClient.builder().serverUrl("https://fauna-android-feature-t-cillum.herokuapp.com/").build();
```

Comment:
Consider adding a helper to instantiate your dependencies, so you can reuse your ApolloClient instance across repositories.